packages/cashier/src/types/stores/index.ts

```diff
@@ -1,6 +1,3 @@
 import { useStore } from '@deriv/stores';
 
 export type TRootStore = ReturnType<typeof useStore>;
```

Comment:
We still need to keep `TRootStore` in the `@deriv/cashier` package as we are overriding the return type of `useStore` hook with `cashier` stores, Later once we managed to replace `cashier` stores with hooks or move them to `@deriv/stores` we should be able to remove this as well.